.lando/adminer.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Provides autologin mode for the Adminer database tool.
+ * @see https://www.adminer.org/en/extension/.
+ */
+
+function adminer_object()
+{
+
+  class AutoLogin extends Adminer
+  {
+
+    public function credentials()
+    {
+      // Server, username and password for connecting to database.
+      return array('database', 'drupal10', 'drupal10');
```

Comment:
Wondering if DB_NAME_DRUPAL etc env variables should be in .env file instead them being .lando.yml This way we could use the env here.

Reply:
Unfortunately, current `adminer` service doesn't support envvars. Tried with these changes:

`.lando.yml`:

```yml
services:
  adminer:
    type: compose
    services:
      image: dehy/adminer
      command: /bin/s6-svscan /etc/services.d
      environment:
        DB_USER_DRUPAL: drupal10
        DB_PASS_DRUPAL: drupal10
        DB_HOST_DRUPAL: database
    run_as_root:
      - /app/.lando/adminer.sh
 ```
 
 `adminer.php`:
 
 ```php
    public function credentials()
    {
      // Server, username and password for connecting to database.
      return array(getenv('DB_HOST_DRUPAL'), getenv('DB_USER_DRUPAL'), getenv('DB_PASS_DRUPAL'));
    }
```

Reply:
But it's easy to update as is during Drupal major upgrade.